smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -209,7 +221,15 @@ async Task<int> OnExecuteAsync()
                                 ? Option.Some(string.IsNullOrEmpty(hostname) ? new HttpUris() : new HttpUris(hostname))
                                 : Option.None<HttpUris>();
 
-                            bootstrapper = new IotedgedLinux(this.BootstrapperArchivePath, credentials, uris, proxy, upstreamProtocolOption, !this.BypassEdgeInstallation);
+                            (bool useConnMan, string connect_management_uri) = this.UseConnMan;
+                            (bool useConnWrkld, string connect_workload_uri) = this.UseConnWrkld;
+                            (bool useLisMan, string listen_management_uri) = this.UseLisMan;
+                            (bool useLisWrkld, string listen_workload_uri) = this.UseLisWrkld;
+                            Option<UriSocks> socks = useConnMan || useConnWrkld || useLisMan || useLisWrkld
```

Comment:
Once address UriSocks constructor comment, you should always call to the same constructor and just use UriSocks instead of Option<UriSocks>.

Reply:
Fixed.